Microsoft.Toolkit.Uwp.UI.Controls/CameraPreview/CameraPreview.xaml

```diff
@@ -0,0 +1,38 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls">
+    <Style x:Name="MessageStyle" TargetType="TextBlock">
+        <Setter Property="FontSize" Value="14"/>
+        <Setter Property="FontWeight" Value="Bold"/>
+        <Setter Property="Foreground" Value="Red"/>
+    </Style>
+    <Style TargetType="local:CameraPreview" >
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:CameraPreview">
+                    <Grid Background="{TemplateBinding Background}">
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto"></RowDefinition>
+                            <RowDefinition Height="Auto"></RowDefinition>
+                        </Grid.RowDefinitions>
+                        <ComboBox x:Name="FrameSourceGroupCombo" Header="Frame Source Group" HorizontalAlignment="Left" Width="Auto">
+                            <ComboBox.ItemTemplate>
+                                <DataTemplate>
+                                    <TextBlock Text="{Binding DisplayName}"></TextBlock>
+                                </DataTemplate>
+                            </ComboBox.ItemTemplate>
+                        </ComboBox>
+                        <StackPanel Grid.Row="1" Orientation="Vertical" Margin="0,10">
+                            <TextBlock x:Name="VideoPreviewTextBlock" Text="Video Preview"  Margin="0,0,0,10"></TextBlock>
+                            <TextBlock x:Name="VideoPreviewErrorTextBlock" Style="{StaticResource MessageStyle}"  Margin="0,0,0,10" Visibility="Collapsed"></TextBlock>
+                            <MediaPlayerElement x:Name="MediaPlayerElementControl" MinWidth="300" Width="400" PosterSource="ms-appx:///Assets/Photos/PreviewUnavailable.png" HorizontalAlignment="Left">
```

Comment:
This will crash if the developer does not have a png at "Assets/Photos/PreviewUnavailable.png" in their app. Either remove this completly or expose this as a property on your control and do TemplateBinding

Reply:
Removed.